lib/Alien/Build/Plugin/Build/Autoconf.pm

```diff
@@ -234,6 +238,67 @@ Some reasonable default flags will be provided.
   $self;
 }
 
+sub _pkgconf_wrapper {
+  my ($self, $build) = @_;
+
+  return if !_win;
+
+  my $pk = File::Which::which ($ENV{PKG_CONFIG})
+        || File::Which::which ('ppkg-config')
+        || File::Which::which ('pkg-config.pl')
```

Comment:
I don't think this will normally be found unless `PATHEXT` is set for .pl which is an unusual configuration I think? I am on the fence about using it.

Reply:
I'll remove it given it seems not to exist anyway.

Reply:
Let's remove pkg-config.pl from the PkgConfig.pm plugin while we are at it.

Reply:
Done in 87fb4bfcbb8b128afda7aded215113952c08ad3c and 230658ec9b7354732a1f980662d3fbd13d031257